components/accordion/index.css

```diff
@@ -46,31 +50,34 @@
 	--spectrum-accordion-item-content-line-height: var(--spectrum-line-height-100);
 
 	/* Colors */
-	--spectrum-accordion-background-color-default: rgba(var(--spectrum-gray-900-rgb), var(--spectrum-background-opacity-default));
-	--spectrum-accordion-background-color-hover: rgba(var(--spectrum-gray-900-rgb), var(--spectrum-background-opacity-hover));
-	--spectrum-accordion-background-color-down: rgba(var(--spectrum-gray-900-rgb), var(--spectrum-background-opacity-down));
-	--spectrum-accordion-background-color-key-focus: rgba(var(--spectrum-gray-900-rgb), var(--spectrum-background-opacity-key-focus));
+	/* @TODO: use opacity tokens when available */
+	--spectrum-accordion-background-color-default: var(--spectrum-transparent-black-25);
+	--spectrum-accordion-background-color-hover: var(--spectrum-transparent-black-100);
+	--spectrum-accordion-background-color-down: var(--spectrum-transparent-black-300);
+	--spectrum-accordion-background-color-key-focus: var(--spectrum-transparent-black-100);
+
+	--spectrum-accordion-item-content-color: var(--mod-accordion-item-content-color, var(--spectrum-body-color));
+	--spectrum-accordion-divider-color: var(--mod-accordion-divider-color, var(--spectrum-gray-200));
+	--spectrum-accordion-focus-indicator-color: var(--spectrum-focus-indicator-color);
 
-	/* Label */
 	--spectrum-accordion-item-header-color-default: var(--spectrum-neutral-content-color-default);
 	--spectrum-accordion-item-header-color-hover: var(--spectrum-neutral-content-color-hover);
 	--spectrum-accordion-item-header-color-down: var(--spectrum-neutral-content-color-down);
 	--spectrum-accordion-item-header-color-key-focus: var(--spectrum-neutral-content-color-key-focus);
 
-	--spectrum-accordion-item-header-disabled-color: var(--spectrum-disabled-content-color);
-
-	/* Focus indicator */
-	--spectrum-accordion-focus-indicator-color: var(--spectrum-focus-indicator-color);
-
-	--spectrum-accordion-min-block-size: max(
-		var(--spectrum-accordion-item-height),
-		calc(var(--mod-accordion-item-header-top-to-text-space, var(--spectrum-accordion-item-header-top-to-text-space)) + var(--mod-accordion-item-header-bottom-to-text-space, var(--spectrum-accordion-item-header-bottom-to-text-space)) + (var(--mod-accordion-item-header-font-size, var(--spectrum-accordion-item-header-font-size)) * var(--mod-accordion-item-header-line-height, var(--spectrum-accordion-item-header-line-height))))
+	/* Calculated minimum block-size that accounts for density spacing changes and/or increases in font size and line-height.
+	   This gives us the block-size of the header with a single line of text. */
+	--spectrum-accordion-item-min-block-size: max(
```

Comment:
holy smokes this is such an intense custom prop! Just for my own sanity, `--spectrum-accordion-item-min-block-size` is the larger value between: 

the defined accordion-item-minimum-height (whether it's a mod or not)

AND

the top padding + the bottom padding + whatever the value is when you multiply the font size by the line height

![frog-wow-scroll](https://github.com/user-attachments/assets/dffd6145-8f17-4b53-9ed8-5017f74c0c4d)

Reply:
Yep, I'm not touching this one if I don't have to.